framework/core/js/src/common/Translator.tsx

```diff
@@ -88,4 +89,16 @@ export default class Translator {
 
     return id;
   }
+
+  /**
+   * Formats the time.
+   * 
+   * The format of the time will be chosen by the following order:
+   * - The format ID defined in current locale.
+   * - The provided fallback format.
+   * - DayJS default format.
+   */
+  format(time: Dayjs, id?: string, fallback?: string): string {
```

Comment:
Maybe use `formatDateTime` to avoid conflicting with `formatSomethingElse` like `formatNumber`.

e.g. 

en: `123.45`
zh: `123.45`
vi: `123,45`

although we may not be using this, we had to consider it.